### PR TITLE
Fix running CI

### PR DESCRIPTION
**Pull request by paladox**
* Bumps MW requirements to 1.39+.
* Bumps SMW requirements to 4.0+.

### CODE REVIEW

**Comment by paladox** [comment]
I don't know why this:

```
There was 1 failure:
1) SCI\Tests\Integration\JSONScript\SemanticCiteJsonTestCaseScriptRunnerTest::testCaseFile with data set "scite-14-fixed-property-query.json" ('/var/www/html/extensions/Sema...y.json')
Failed "#0 non display formatted query output"
==== (actual) ====
<div class="mw-parser-output"><p>{{#ask:<span class="smw-highlighter" data-type="4" data-state="inline" data-title="Warning" title="&quot;~*M114.559054&quot; is an invalid value for the DOI type."><span class="smwtticon warning"></span><span class="smwttcontent">"~*M114.559054" is an invalid value for the DOI type.</span></span>|?DOI|format=table}}
</p></div>
==== (StringContains expected) ====
[ <td class="DOI smwtype_sci_doi"><span class="plainlinks"><a rel="nofollow" class="external text" href="https://doi.org/10.1074%2Fjbc.m114.559054">10.1074/jbc.m114.559054</a></span></td> ]
Failed asserting that 0 matches expected 1.
/var/www/html/extensions/SemanticCite/extensions/SemanticMediaWiki/tests/phpunit/Utils/Validators/StringValidator.php:82
/var/www/html/extensions/SemanticCite/extensions/SemanticMediaWiki/tests/phpunit/Utils/Validators/StringValidator.php:33
/var/www/html/extensions/SemanticCite/tests/phpunit/Integration/JSONScript/SemanticCiteJsonTestCaseScriptRunnerTest.php:214
/var/www/html/extensions/SemanticCite/tests/phpunit/Integration/JSONScript/SemanticCiteJsonTestCaseScriptRunnerTest.php:149
/var/www/html/extensions/SemanticCite/extensions/SemanticMediaWiki/tests/phpunit/JSONScriptTestCaseRunner.php:231
/var/www/html/tests/phpunit/MediaWikiIntegrationTestCase.php:500
/var/www/html/extensions/SemanticCite/extensions/SemanticMediaWiki/tests/phpunit/SMWIntegrationTestCase.php:205
FAILURES!
```

fails?

Any ideas on how to fix @gesinn-it-gea / @gesinn-it-ilm ?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,27 +17,20 @@ jobs:
...
             experimental: false
           - mediawiki_version: '1.39'
             smw_version: dev-master
             php_version: 8.1
             database_type: mysql
-            database_image: "mariadb:latest"
-            coverage: false
-            experimental: true
+            database_image: "mariadb:11.2"
+            coverage: true
```
Comment (gesinn-it-ilm) [inline .github/workflows/ci.yml:32]: Coverage won't work with MW 1.39. It will work with MW 1.40.

Comment (paladox) [reply .github/workflows/ci.yml:39]: Thanks, done.

**Comment by paladox** [comment]
>   [Symfony\Component\Console\Exception\CommandNotFoundException]  
  Command "test-coverage" is not defined.

File: tests/phpunit/Integration/JSONScript/TestCases/scite-14-fixed-property-query.json [hunk 2]
```diff
@@ -19,6 +19,9 @@
 		{
 			"about": "#0 non display formatted query output",
 			"subject": "Example/Query/1",
+			"skip-on": {
+				"mediawiki": [ ">1.38.x", "Check for MW 1.39+" ]
```
Comment (JeroenDeDauw) [inline tests/phpunit/Integration/JSONScript/TestCases/scite-14-fixed-property-query.json:23]: Does not seem like a good message/description. "Skip on MW 1.39 and above" is a lot clearer, and you could also add something about why the skip is in place.

Comment (paladox) [reply tests/phpunit/Integration/JSONScript/TestCases/scite-14-fixed-property-query.json]: Thanks, done.

**Pull request merged**